modules/git/Git_basics.md

```diff
@@ -338,9 +332,9 @@ git commit
   * lesson 02 on git
   ~~~
   
-* After you wrote and saved the message ("save" in `atom`, `^O` in
+* After you wrote and saved the message ("save" in `vs code`, `^O` in
```

Comment:
If we refer to the command itself then it would be `code` (no "vs"). If we refer to the name then "vs code" or to be clearer use the official name "Visual Studio Code".